### DISCUSSION TITLE
Prevent the master from seeing the recently accessed devices of the slave

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#6464] [General]
privacy issues

**Comment by 21pages** [comment]
![rustdesk_CNQDcjDyF0](https://github.com/rustdesk/rustdesk/assets/14891774/a3347a38-e8a4-4ceb-b130-d8ca58a8e52c)

**Reply by ghost** [reply]
I know this can be hidden manually, but I hope it can be hidden so that the master cannot see it every time it is controlled.
